components/ui/ModalCreation.vue

```diff
@@ -69,6 +83,45 @@ import CheckIcon from '~/assets/images/utils/right-arrow.svg'
 import Modal from '~/components/ui/Modal.vue'
 import Chips from '~/components/ui/Chips.vue'
 
+const messages = defineMessages({
+  modalTitle: {
+    id: 'modal.create-project.title',
+    defaultMessage: 'Create a project',
+  },
+  modalDescription: {
+    id: 'modal.create-project.description',
+    defaultMessage: 'New projects are created as drafts and can be found under your profile page.',
+  },
+  modalDescriptionNote: {
+    id: 'modal.create-project.description.note',
+    defaultMessage: 'Some project metadata like summary & description must be written in English.',
```

Comment:
Never use ampersand in sentence form.

```suggestion
    defaultMessage: 'Some project metadata, such as the summary and description, must be written in English.',
```